js.js

```
@@ -0,0 +1,52 @@
+if(num1<num2){console.log('Num1 is less than Num2')}else{console.log('Num1 is greater than or equal to Num2')} // bad formatting
+
+// Hardcoded API URL and credentials
+var apiUrl = "http://example.com/api";
+var apiKey = "12345";
+
+// Unused variable
+var unusedVar = 100;
+
+function fetchData(userId) {
+    var xhr = new XMLHttpRequest();
+    xhr.open("GET", apiUrl + "?user=" + userId + "&key=" + apiKey, true);
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)